homework_21/homework_21.txt

```diff
@@ -0,0 +1,8 @@
+1 - SELECT * FROM users WHERE age > 30
+
+2 - SELECT COUNT(*) FROM users WHERE age > 30
+
+3 - SELECT age, COUNT(*) FROM users GROUP BY age
```

Comment:
You can use an alias for the aggregation function here. The name will be displayed then as a column name.
`select age, COUNT(*) as users_count ...`